From 6a3fed65b85a4bc592265dde0d74d6d7a3185543 Mon Sep 17 00:00:00 2001
From: Richard Kiene <richard.kiene@joyent.com>
Date: Tue, 31 Oct 2017 19:34:14 +0000
Subject: [PATCH] CNS-216 Only fetch CNs that are setup Reviewed by: Todd
 Whiteman <todd.whiteman@joyent.com> Approved by: Todd Whiteman
 <todd.whiteman@joyent.com>

---
 lib/cn-filter.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/cn-filter.js b/lib/cn-filter.js
index c0231e7..7716bd5 100644
--- a/lib/cn-filter.js
+++ b/lib/cn-filter.js
@@ -61,7 +61,7 @@ util.inherits(CNFilter, stream.Transform);
 
 CNFilter.prototype.updateServers = function (cb) {
 	var self = this;
-	self.client.get('/servers?extras=status',
+	self.client.get('/servers?setup=true&extras=status',
 	    function (err, req, res, objs) {
 		if (err) {
 			self.log.warn(err,
-- 
2.21.0

